modules/client/src/addresslistVoting/internal/client/methods.ts

```diff
@@ -75,6 +68,15 @@ import {
   UNSUPPORTED_PROPOSAL_METADATA_LINK,
 } from "../../../client-common/constants";
 import { AddresslistVotingClientEncoding } from "./encoding";
+import {
+  AddresslistVotingPluginPrepareInstallationParams,
+  AddresslistVotingProposal,
+  AddresslistVotingProposalListItem,
+} from "../../types";
+import {
+  SubgraphAddresslistVotingProposal,
+  SubgraphAddresslistVotingProposalListItem,
+} from "../types";
```

Comment:
Isn't it a bit confusing to have two `types` files like this?
IMO the subgraph only types could be in a dependencies (or similar) file

Reply:
We have the internal types that are only used by the sdk, and the ones outside external which are all exported. Maybe moving the subgraph types to its own package makes sense, but we will still have some internal types that we dont want to export